src/rgw/setup/radosgw_start

```diff
@@ -3,6 +3,10 @@
 INDEX=$1
 CONFIG_FILE=$2
 LOG_FILE=$3
+rgw_crash_dir=$4
```

Comment:
rename to rgw_cwd

And update all its references in comments also

Reply:
done